packages/wrangler/src/pipelines/index.ts

```diff
@@ -386,7 +388,8 @@ export function pipelines(pipelineYargs: CommonYargsArgv) {
 					pipelineConfig.destination.compression.type = args.compression;
 				}
 				if (args["batch-max-mb"]) {
-					pipelineConfig.destination.batch.max_mb = args["batch-max-mb"];
+					pipelineConfig.destination.batch.max_mb =
```

Comment:
Why is this bytes if the API property is called `max_mb`?

Reply:
Good catch, let me update it!